types/feedme/feedme-tests.ts

```diff
@@ -0,0 +1,63 @@
+import * as FeedMe from "feedme";
```

Comment:
Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

Reply:
Interesting, I never knew the subtle differences there. Saying that, it doesn't seem to throw any errors when compiling in tsc, and when using the module with these definitions in a real project (using the import syntax identical to the test file) seems to work just fine too.

So, if the compiler doesn't throw any errors and the feedme package works as described by the definitions and test files, then what's the problem here? I can add a CommonJS import syntax to the test file too, if that helps, which I've already tested and works fine with the definitions.

Reply:
> then what's the problem here? 

It's manifestly illegal ES6 code?

TypeScript will almost certainly break this behavior at some point in the future, or at the very least we'll enforce that this is done correctly on DefinitelyTyped. You should remove the `import *` from the testcase.

Reply:
Can't argue with that.

A "please" would have been nice, though. We're all still learning, I was only interested as to why the compiler didn't pick it up as a syntax error.

Anyway, I'll fix that test case in the morning and push the changes.

Reply:
Thanks!